COPYING

```diff
@@ -295,8 +295,7 @@ the "copyright" line and a pointer to where the full notice is found.
 
     This program is free software; you can redistribute it and/or modify
     it under the terms of the GNU General Public License as published by
-    the Free Software Foundation; either version 2 of the License, or
-    (at your option) any later version.
+    the Free Software Foundation; either version 2 of the License.
```

Comment:
Or to be consistent with the other change `under the terms of version 2 of the GNU General Public License as published by the Free Software Foundation`.